Source/OpalGraphics/CGImageDestination-private.h

```diff
@@ -1,22 +1,22 @@
-#include "CoreGraphics/CGImageDestination.h"
-#import <Foundation/NSArray.h>
-#import <Foundation/NSSet.h>
-
-@interface CGImageDestination : NSObject
-{
-}
-
-+ (void) registerDestinationClass: (Class)cls;
-+ (NSArray*) destinationClasses;
-+ (Class) destinationClassForType: (NSString*)type;
-
-+ (NSArray *)typeIdentifiers;
-- (id) initWithDataConsumer: (CGDataConsumerRef)consumer
-                       type: (CFStringRef)type
-                      count: (size_t)count
-                    options: (CFDictionaryRef)opts;
-- (void) setProperties: (CFDictionaryRef)properties;
-- (void) addImage: (CGImageRef)img properties: (CFDictionaryRef)properties;
-- (bool) finalize;
-
+#include "CoreGraphics/CGImageDestination.h"
+#import <Foundation/NSArray.h>
+#import <Foundation/NSSet.h>
+
+@interface CGImageDestination : NSObject
+{
+}
+
++ (void) registerDestinationClass: (Class)cls;
++ (NSArray*) destinationClasses;
++ (Class) destinationClassForType: (NSString*)type;
+
++ (NSArray *)typeIdentifiers;
```

Comment:
The formatter doesn't seem to standardize the space before the `*`:
```
+ (NSArray*) destinationClasses;
+ (NSArray *)typeIdentifiers;
```
I believe `(NSArray *)` is the standard format in GNUstep, we could probably do a regex fix if the formatter can't fix this.